src/claimList.js

```diff
@@ -3,49 +3,256 @@ import {
+                              claim.type,
+                              claim.status
+                            )
+                          )}
+                        `}
+                  </div>`
+                )
+                .join("")}
+            </div>`
+          : html` <div
+              style="background-color: #fff; border: 1px solid #DFE1E2; border-radius: 4px; padding: 32px; font-size: 16px; line-height: 24px"
+            >
+              <b>No recent claims on file.</b> We haven't received a claim from
+              you in the past 12 months. If you recently applied, expect an
+              update within the next couple weeks.
+            </div>`}
+      </div>
+      ${oldClaims.length > 0
+        ? html` <h2 style="font-size: 22px; line-height: 32px; margin: 0">
+              Older
+            </h2>
+            <hr
+              style="
+                margin: 0;
+                border: none;
+                border-top: 1px solid #dfe1e2;
+                margin-bottom: 8px;
+              "
+            />
+            <div
+              style="text-transform: uppercase; font-size: 13px; line-height: 16px; margin-bottom: 16px"
+            >
+              started more than a year ago
+            </div>
+            ${oldClaims
+              .map(
+                (claim) => html`
+                  <button
+                    style="	background: none;
+                      color: inherit;
+                      font: inherit;
+                      cursor: pointer;
+                      outline: inherit; 
+                      display: flex;
+                      align-items: center;
+                      border: 0.5px solid #A9AEB1;
+                      padding: 16px 32px;
+                      background-color: #fff;
+                      border-radius: 4px;
+                      gap: 8px;
+                      text-align: left;
+                      margin-bottom: 8px;
+                      width: 100%;
+                      justify-content: space-between"
+                  >
+                    <div>
+                      <b>Claim for ${getClaimTypeContent(claim.type)}</b>,
+                      started ${getFormattedDate(claim.date)}
+                    </div>
+                    <img src="./assets/navigate_next.svg" alt="" />
+                  </button>
+                `
+              )
+              .join("")}`
+        : ""}
+      <div style="margin-bottom: 8px; margin-top: 44px">
+        ${RETURN_TO_TOP_LINK}
+      </div>
+    </div>
+    ${FOOTER_HTML}`;
+  root?.append(newContainer);
 }
 
 function logView(allClaims) {
   const recentClaims = allClaims
-    .map((claim) => claim.claimDate)
+    .map((claim) => claim.date)
     .filter((date) => {
-      const claimDateObj = new Date(date);
+      const dateObj = new Date(date);
       const now = new Date();
       const sixMonthsAgo = new Date().setMonth(now.getMonth() - 6);
```

Comment:
Pebble: Internally we are considering "recent" to be mean anything in the last 6 months, but in the UI it would be anything within 12 months. Not sure if they are meant to be aligned or if it's intentional.

![Screenshot_2024-05-03_at_12_49_13_PM](https://github.com/newjersey/paid-leave-claim-status/assets/1824854/80986423-bbc5-4313-bee5-bd8e148ab8fd)

Reply:
Ah yes, great eye... this one's tricky. so we originally started logging recent claims as "6 months" for our own understanding, but we now want to display for users "1 year ago" based on that 6 month data. I was unsure if it'll be confusing to switch the metric logic behind the scenes, and had kept it as is since we don't actually need them to be aligned. Perhaps it makes sense to be changing it now though so that we have insight into how many cards people are seeing on the UI in each section?

Reply:
I don't think they have to be aligned, just wanted to sanity check that this was intended. I think the internal concept of recent should depend on what we actually want to track. I actually think it could be interesting to just return some sort of object in the logged event that has something like <6 months, <12 months, > 12 months so that we could have more flexibility in showing these metrics over time.

Reply:
Also as a tangent, I also wonder if there's an additional/different way to fire the analytics that considers their actual statuses? Because maybe it matters less than that they have 5 ongoing claims if only 1 is undetermined or something? Again I guess it matters on what we want to track in dashboards - is it helpful to track that the amount of recent/total claims? Is it more important to track recent undetermined claims (or any other subset of claims)?